Button/Button.jsx

```diff
@@ -42,5 +48,14 @@ class ButtonIcon extends Icon {
   }
 }
 
+const _generateThemeClass = prop => {
```

Comment:
i guess there are more components that can use this e.g. fab.

shall we move this into a util instead?

Reply:
Just create a `utils` folder for it?

Reply:
`utils.js/ThemeUtils.js` may be.